commit d91f9583f6a3b0d40912ba35a71534255500b371
Author: Alexis Christoforides <alexis@thenull.net>
Date:   Fri Mar 6 15:30:34 2015 -0500

    Remove demos from build, they fail when using a gdk-pixbuf that is staged.

diff --git a/Makefile.am b/Makefile.am
index 8e3a2f1..7288db2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,7 +1,7 @@
 ## Makefile.am for GTK+
 include $(top_srcdir)/Makefile.decl

-SRC_SUBDIRS = gdk gtk modules demos tests perf
+SRC_SUBDIRS = gdk gtk modules tests perf
 SUBDIRS = po po-properties $(SRC_SUBDIRS) docs m4macros build

 # require automake 1.4
